docs/tutorials/shor.ipynb

```diff
@@ -1201,21 +1062,11 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 21,
+   "execution_count": null,
    "metadata": {
     "id": "1RN4SqfDYAsh"
    },
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "Factoring n = pq = 184573\n",
```

Comment:
What is the runtime of this factorization? If it's more than ~10 seconds, we may want to keep the outputs here to prevent delays in CI tests.

Reply:
It's fast! (due to classical order finder)

```
CPU times: user 2 µs, sys: 1e+03 ns, total: 3 µs
Wall time: 5.72 µs
Factoring n = pq = 184573
p = 379
q = 487
```